cluster/gce/config-test.sh

```diff
@@ -287,8 +287,19 @@ if [[ -n "${GCE_ALPHA_FEATURES:-}" ]]; then
   PROVIDER_VARS="${PROVIDER_VARS:-} GCE_ALPHA_FEATURES"
 fi
 
-# If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
-ADMISSION_CONTROL="${KUBE_ADMISSION_CONTROL:-Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota}"
+# Optional: enable the PodSecurityPolicy admission contrller.
+ENABLE_POD_SECURITY_POLICY="${ENABLE_POD_SECURITY_POLICY:-false}"
```

Comment:
only use falls back to empty string, no need to assign to ENABLE_POD_SECURITY_POLICY here

Reply:
Done.